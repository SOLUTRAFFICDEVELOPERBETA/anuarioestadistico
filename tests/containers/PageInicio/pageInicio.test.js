import React from 'react'
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import PageInicio from '../../../containers/pageInicio';
import { fields } from '../../fixtures';

describe('Prueba en <PageInicio/>', () => {
    const wrapper = shallow(
        <PageInicio
            fields={fields}
        />
    )
    test('debe mostrarse correctamente', () => {
        expect(wrapper).toMatchSnapshot();
    })

    test('la imagen debe mostrarse correctamente', () => {
        const img = wrapper.find('img');
        expect(img.prop('src')).toBe(fields[4].value.url);
    })
})