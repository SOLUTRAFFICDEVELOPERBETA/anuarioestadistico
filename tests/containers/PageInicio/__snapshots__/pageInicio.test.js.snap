// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prueba en <PageInicio/> debe mostrarse correctamente 1`] = `
<Styled(MuiBox)
  paddingBottom={2}
  paddingLeft={1}
  paddingRight={1}
  style={
    Object {
      "background": "#f2f4f7",
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    spacing={3}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <Styled(MuiBox)>
        <WithStyles(ForwardRef(Typography))
          color="textPrimary"
          style={
            Object {
              "fontWeight": 700,
              "marginBottom": "0.5rem",
            }
          }
          variant="h5"
        >
          Este es el primer valor de prueba
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          align="justify"
          color="textSecondary"
          component="p"
          style={
            Object {
              "marginBottom": "0.5rem",
            }
          }
        >
          Este es el segundo valor de prueba
        </WithStyles(ForwardRef(Typography))>
        <WithStyles(ForwardRef(Typography))
          align="justify"
          color="textSecondary"
          component="p"
          style={
            Object {
              "marginBottom": "0.5rem",
            }
          }
        >
          Este es el tercero valor de prueba
        </WithStyles(ForwardRef(Typography))>
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <Styled(MuiBox)>
        <img
          alt="imgmovidad"
          height={400}
          src="https://www.valladolid.mx/assets/img/default.jpg"
          width={600}
        />
      </Styled(MuiBox)>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</Styled(MuiBox)>
`;
